js/main.js

```diff
@@ -208,7 +210,7 @@ var attentionScum = function(vocabulary) {
 var buildQuote = function(vocabulary) {
     var quote = '';
 
-    if (Math.random() < 0.85) {
+    if (_.random(9) < 8) { // 80% chance
```

Comment:
Augh! No, you were right the first time. I was sorta hemming and hawwing until I saw this. And it doesn't even get you 85%, that would have to be like `random 19 < 17`?. Horrible. I get why it's a float between 0 and 1 now.